policy/modules/services/container.te

```diff
@@ -317,6 +317,7 @@ allow container_domain container_ro_file_t:sock_file read_sock_file_perms;
 fs_tmpfs_filetrans(container_domain, container_tmpfs_t, { dir file fifo_file lnk_file sock_file })
 manage_dirs_pattern(container_domain, container_tmpfs_t, container_tmpfs_t)
 mmap_manage_files_pattern(container_domain, container_tmpfs_t, container_tmpfs_t)
+exec_files_pattern(container_domain, container_tmpfs_t, container_tmpfs_t)
```

Comment:
Is the full `execute_no_trans` needed or only `execute`?

Reply:
Yes, only `execute` is needed. Changing this accordingly.

Reply:
In that case, you want `mmap_exec_files_pattern` instead of `can_exec`.